packages/odyssey-storybook/src/components/odyssey-labs/DatePicker/DatePicker.stories.tsx

```diff
@@ -12,11 +12,79 @@
 
 import { useMemo, useState } from "react";
 import type { Meta, StoryObj } from "@storybook/react";
+import { screen, userEvent, waitFor, within } from "@storybook/testing-library";
+import { expect } from "@storybook/jest";
 
 import { DatePicker, DatePickerProps } from "@okta/odyssey-react-mui/labs";
 
 import { fieldComponentPropsMetaData } from "../../../fieldComponentPropsMetaData";
 import { MuiThemeDecorator } from "../../../../.storybook/components";
+import { axeRun } from "../../../axe-util";
+import type { PlaywrightProps } from "../../odyssey-mui/storybookTypes";
+
+const findInputElement = async ({
+  canvasElement,
+  step,
+}: PlaywrightProps<DatePickerProps>) => {
+  await step("Find Input", async () => {
+    const canvas = within(canvasElement);
+    const input = canvas.getByPlaceholderText("MM/DD/YYYY") as HTMLInputElement;
+    await userEvent.click(input);
+  });
+};
+
+const clickCalendarButton = async ({
+  canvasElement,
+  step,
+}: PlaywrightProps<DatePickerProps>) => {
+  await step("Click Calendar Button", async () => {
+    const canvas = within(canvasElement);
+    const buttonElement = canvas.getByLabelText("Choose date");
+    await userEvent.click(buttonElement);
+  });
+};
+
+const openCalendarAndSelectDate =
+  ({ canvasElement, step }: PlaywrightProps<DatePickerProps>) =>
+  async (actionName: string) => {
+    await step("select date", async ({ args }) => {
+      // I don't know why this doesn't work without this step
+      await findInputElement({ canvasElement, step });
+      await clickCalendarButton({ canvasElement, step });
+      await waitFor(() => {
+        axeRun(actionName);
+      });
+      const calendar = await screen.findByRole("dialog");
+      const calendarCanvas = within(calendar);
+      const date = calendarCanvas.getByText("14");
+      await userEvent.click(date);
+      await expect(args.onCalendarDateChange).toHaveBeenCalledTimes(1);
+    });
+  };
+
+const openCalendarAndCheckForDisabledDate =
+  ({
+    canvasElement,
+    step,
+    buttonText,
+  }: PlaywrightProps<DatePickerProps> & { buttonText: string }) =>
+  async (actionName: string) => {
+    await step("attempt to select disabled date", async ({ args }) => {
+      // I don't know why this doesn't work without this step
+      await findInputElement({ canvasElement, step });
+      await clickCalendarButton({ canvasElement, step });
```

Comment:
The calendar dropdown doesn't show up unless you have the input focused right? Maybe that's why?

We didn't focus on the input though.